modules/rbac-resource-definitions.adoc

```diff
@@ -6,15 +6,21 @@
 = Resource definitions
 
 [role="_abstract"]
-{product-title} includes multiple resources.
+{product-title} includes many resources.
 The following table lists the resources and describes the actions that users can perform with the `read` or `write` permission.
 
+[NOTE]
+====
+* To prevent privilege escalation, when you create a new token, your role's permissions limit you. For example, if you only have `read` permission for the Integration resource, you cannot create a token with `write` permission.
+* If you want a custom role to create tokens for other users to use, you must assign the required permissions to that custom role.
```

Comment:
I think the text is great, however we can also create an additional link IMO to the two following places:
`operating/manage-user-access/configure-short-lived-access` and `cli/using-the-roxctl-cli#using-an-authentication-provider-to-authenticate-with-roxctl_using-roxctl-cli`.

Let me try and give a little bit of history around this and what I'd propose the section to include:
Previously, we only had API tokens with which users could authenticate against ACS _when they were not using it interactively_, meaning when using `roxctl` or creating custom automation scripts.

We have now _two_ alternatives for API tokens:
- short-lived tokens, which can be used when accessing ACS non-interactively (e.g. in CI); this can be used for both scripts and roxctl.
- roxctl central login - which can be used when accessing ACS with roxctl _but_ having a "human" call ACS; i.e. this does not work with non-interactive access in e.g. CI.

I think the best place to point out a) about the alternatives of authentication and b) the privilege escalation would be here: `configuration/configure-api-token` when one creates a new API token. Otherwise, this information _may_ get lost here I feel like, since creating a permission set and creating a new API token are two different things IMO.

Reply:
**praise:** 👏 Thank you @dhaus67 I'll update the description and add it to suggested sections.